grass7/raster/r.centroids/r.centroids.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+############################################################################
+#
+# MODULE:    r.centroids
+#
+# AUTHOR(S): Caitlin Haedrich (wrapper for r.volume, caitlin dot haedrich gmail com)
+
+#
+# PURPOSE:   Creates vector map of centroids from a raster of "clumps"; r.clumps
+#            creates "clumps" of data.
+#
+# COPYRIGHT: (C) 1997-2016 by the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+#
+#############################################################################
+
+#%module
+#% description: Creates vector map of centroids from raster of "clumps"
+#% keyword: raster
+#% keyword: centroid
+#% keyword: clumps
+#% keyword: vector
+#% keyword: centerpoint
+#%end
+
+#%option G_OPT_R_INPUT
+#% key: input
+#% description: raster of clumps, cluster of same-valued pixels
+#% required: yes
+#%end
+
+#%option G_OPT_V_OUTPUT
+#% key: output
+#% description: name of output vector
```

Comment:
Uppercase here as well

Reply:
Actually, you may be able to delete the description line altogether, there may be a default one.

Reply:
I deleted the description lines in hopes there a default one.